src/components/Header.tsx

```diff
@@ -0,0 +1,46 @@
+import { forwardRef, Dispatch } from 'react';
+import cn from 'classnames';
+
+import { Todo } from '../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  addTodo: (event: React.FormEvent<HTMLFormElement>) => void;
```

Comment:
You are exposing unnecessary inner details 

The much cleaner option is 
```suggestion
  addTodo: (title: string) => void;
```